src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -626,8 +626,10 @@ describe('ReactDOMComponent', () => {
     }
 
     beforeEach(() => {
-      var ReactDefaultInjection = require('ReactDefaultInjection');
-      ReactDefaultInjection.inject();
+      var ReactDOMInjection = require('ReactDOMInjection');
+      ReactDOMInjection.inject();
+      var ReactDOMStackInjection = require('ReactDOMInjection');
```

Comment:
Ah. Thanks. Updated. But I guess I don't technically need it to pass this test. Will revisit when we try to make this test work with Fiber.